src/utils/expenses.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const isValidData = (data, method) => {
+  const {
+    spentAt,
+    title,
+    amount,
+    category,
+  } = data;
+
+  if (method === 'POST') {
+    return typeof spentAt === 'string'
+    && typeof title === 'string'
+    && typeof category === 'string'
+    && spentAt.length > 0
+    && title.length > 0
+    && category.length > 0
+    && typeof amount === 'number';
+  }
+
+  return isValidStringField(spentAt)
+    && isValidStringField(title)
+    && isValidStringField(category)
+    && isValidNumericField(amount)
+    && isValidNumericField(amount);
+};
+
+const isValidStringField = (field) => {
+  let isValid = true;
+
+  if (typeof field !== 'undefined') {
+    isValid = typeof field === 'string' && field.length > 0;
+  }
+
+  return isValid;
+};
```

Comment:
```suggestion
const isValidStringField = field => (
  field && typeof field === 'string' && field.length,
);
```
Use short notation and keep it clear.

Reply:
Nice))